src/app/Components/Button.svelte

```diff
@@ -0,0 +1,122 @@
+<script lang="ts">
+  import { beforeUpdate } from "svelte";
+
+  // props
+  export let color: string = "grey"; // optional
+  export let isOutlined: boolean = false; // optional
+  export let hasDropShadow: boolean = false; // optional
+  export let size: string = "medium"; // optional
+  export let text: string = ""; // required
+  export let subtext: string = ""; // optional
+  export let callbackFunction: any = null; // required
```

Comment:
Since consumers of this component will use need to set this property, I would rename it to what it means:

```suggestion
  export let onClick: (ev: MouseEvent) => void;
```

That way, users of this will use it like this:

```svelte
<Button onClick={() => console.log("clicked")}>Click me!</Button>
```